packages/switchToggle/src/Default/Default.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { Form, ThemeProvider } from 'react-bootstrap';
+import {
+  SwitchToggleDefaultProps,
+  SwitchToggleContainerProps,
+} from './Default.types';
+import * as styleMod from './styles.module.scss';
+import { createLocalProps } from '@owlui/utils';
+
+export const Component = (props: SwitchToggleDefaultProps) => {
+  const baseClass = 'switchToggle';
+  const { switchtoggle } = props;
+  const prefix = props.prefix || '';
+
+  const [toggle, setToggle] = React.useState<boolean>(false);
+
+  const locals: SwitchToggleDefaultProps = createLocalProps(
+    props,
+    {
+      module: styleMod,
+      classes: {
+        base: baseClass,
+        prefix: prefix,
+        optionals: [
+          {
+            fields: ['theme'],
+            value: 'theme',
+          },
+          {
+            fields: ['appearance'],
+            value: 'Appearance',
+          },
+          {
+            fields: ['size'],
+            value: 'Size',
+          },
+        ],
+      },
+    },
+    [
```

Comment:
This is the removable array, all items in this array will be removed from props, so you should only add what you don't want to pass to the component in the `locals` object.

You added some props that are default in the `<Form.Check>` component, such as the `label`. Also, there's no appearance in the component so you can get rid of all reference of appearance.

I believe you should keep only "our" props here, like `['prefix', 'theme', 'Size']`.